subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ThirdPartyPluginsSmokeTest.groovy

```diff
@@ -269,7 +276,7 @@ class ThirdPartyPluginsSmokeTest extends AbstractSmokeTest {
         given:
         buildFile << """
             plugins {
-                id "org.xtext.xtend" version "1.0.5"
+                id "org.xtext.xtend" version "1.0.17"
```

Comment:
We should also try to use the latest xtend lib: https://bintray.com/bintray/jcenter/org.eclipse.xtend%3Aorg.eclipse.xtend.lib/2.11.0